utils/net.py

```diff
@@ -43,13 +42,7 @@ def my_ip_address(http=False):
 
     """
     # Allow for DockerBot to set outside IP
-    try:
-        # Check the environment first
-        return os.environ['CFME_MY_IP_ADDRESS']
-    except KeyError:
-        # Fall back to having an appliance tell us what it thinks our IP
-        # address is
-        return store.current_appliance.ssh_client().client_address()
+    return store.current_appliance.my_ip_address
```

Comment:
Take a look at `store.my_ip_address` (specifically the part where it's lazycached).

With regard to moving the behavior onto `IPAppliance`, we get the test runner IP address from the appliance, so it makes sense to have the behavior there. I don't fully recall, but I also think the move prevented a circular import.